klayout_package/python/scripts/simulations/elmer/run_helpers.py

```diff
@@ -329,16 +330,33 @@ def run_elmer_solver(json_data: dict[str, Any], exec_path_override: Path | str |
     )
 
 
-def run_paraview(result_path: Path | str, n_processes: int, exec_path_override: Path | str | None = None):
+def run_paraview(
+    result_path: Path | str, n_processes: int, exec_path_override: Path | str | None = None, cross_section: bool = False
```

Comment:
```suggestion
    result_path: Path | str, exec_path_override: Path | str | None = None, cross_section: bool = False
```
I think n_processes is no longer used in this function